ingredient/ingredient-function-app/create-ingredient-quickstart.md

```diff
@@ -0,0 +1,79 @@
+# Quick Start: Create a new ingredient that deploys an ARM template
```

Comment:
Remove this file, doesn't need to be in an ingredient project

Reply:
removed